src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void getInfo();
```

Comment:
```suggestion
    public void draw();
```
<img width="846" alt="Знімок екрана 2023-01-21 о 13 56 34" src="https://user-images.githubusercontent.com/41166578/213865758-82dbb238-1277-4461-9922-8f14b468b731.png">